akka-remote-tests/src/multi-jvm/scala/akka/remote/RemoteNodeShutdownAndComesBackSpec.scala

```diff
@@ -118,8 +118,8 @@ abstract class RemoteNodeShutdownAndComesBackSpec
         watch(subjectNew)
 
         subjectNew ! "shutdown"
-        fishForMessage(5.seconds) {
-          case _: ActorIdentity       ⇒ false
+        receiveWhile(5.seconds) {
+          case _: ActorIdentity       ⇒ true
           case Terminated(subjectNew) ⇒ true
```

Comment:
I think this will always take 5 seconds now. We would like to continue earlier if we receive the `Terminated`, because then we know that the other side is done.

receiveWhile:

     * Receive a series of messages until one does not match the given partial
     * function or the idle timeout is met (disabled by default) or the overall
     * maximum duration is elapsed or expected messages count is reached.
     * Returns the sequence of messages.

If I understand that correctly it should work if you remove the `case Terminated` (and write a comment).
but you better try it out, so that I'm not wrong.